packages/components/package.json

```diff
@@ -0,0 +1,52 @@
+{
+    "name": "components",
+    "private": true,
+    "version": "0.0.0",
+    "type": "module",
+    "files": [
+        "dist",
+        "tailwind.config.js"
+    ],
+    "main": "./dist/component-library.cjs",
+    "module": "./dist/component-library.js",
+    "exports": {
+        "./content": {
+            "import": "./.contentlayer/generated/index.mjs",
+            "types": "./.contentlayer/generated/index.d.ts"
+        },
+        ".": {
+            "import": "./dist/lib/component-lib.js",
+            "types": "./dist/lib/component-lib.d.ts"
+        },
+        "./styles": "./dist/lib/index.css",
+        "./tailwind": "./tailwind.config.js"
+    },
+    "scripts": {
+        "dev": "tsc && concurrently 'vite build --watch' 'contentlayer dev'",
+        "build": "tsc && pnpm build:lib && contentlayer build ",
+        "build:lib": "vite build",
+        "preview": "vite preview"
+    },
+    "sideEffects": [
+        "**/*.css"
+    ],
+    "devDependencies": {
+        "@types/react": "^18.2.51",
+        "typescript": "^5.2.2",
+        "vite": "^5.0.8",
+        "vite-plugin-lib-inject-css": "^1.3.0"
```

Comment:
Do we really want to have the css file injected via JS? I would prefer a different export as it is in the current fondue version